### ISSUE TITLE
Full-Spec-Plot Index Error

### ISSUE THREAD

**Issue by MrFunBarn** [MrFunBarn/chiron-spec-tools#1]
Error when generating full-spec plot with more than one order normalized. 

Error message:

``` bash
Traceback (most recent call last):
  File "/usr/bin/normalize-chiron", line 115, in <module>
    spect.whole_spec_plot(25,100)
  File "/usr/lib/python2.7/site-packages/interactivenorm.py", line 706, in whole_spec_plot
    middle_index = int(len(self.sm[order][:,0]) // 2)
TypeError: 'int' object has no attribute '__getitem__'
```

**Comment by MrFunBarn** [comment]
I mixed up th issue number while commiting, really fixed #2. This issue is still open.

**Comment by MrFunBarn** [comment]
Whole spec plot completes as expected, if it has allready been run and the state pickled. very curious. I think that the problem lies somewhere in order handling and passing to smooth. Also works fine if no orders are normalized.
